index.html

```diff
@@ -27,19 +34,18 @@
   </head>
   <body>
     <div class="first-page">
-      <header class="header">
-        <div class="container">
-          <a href=""><img src="images/MyLogo.png" alt="Logo" class="logo" /></a>
-          <nav class="item-2">
-            <img src="images/Hamburguer.png" alt="Menu" />
-            <ul class="ul-menu">
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
-            </ul>
-          </nav>
-        </div>
+      <header class="navbar">
+        <a href="#" class="logo"><img src="images/MyLogo.png" alt="" /></a>
+        <div id="icon" class="icon fas fa-bars"></div>
```

Comment:
**You made a great effort in creating this webpage but there are tiny issues that need to be fixed.** 

- [ ] I can see the `Hamburger` button on your desktop version of your project. The requirements says The `Hamburger` button should not be displayed on the desktop version:

<img width="440" alt="Screen Shot 2022-01-25 at 7 34 51 AM" src="https://user-images.githubusercontent.com/69011963/150903070-a4854e23-f141-4260-a531-20f709390c06.png">